Metalama.Framework/Advising/IAdviceFactory.cs

```diff
@@ -179,7 +184,7 @@ public interface IAdviceFactory
         /// <param name="tags">An optional opaque object of anonymous type passed to the template method and exposed under the <see cref="meta.Tags"/> property of the
         ///     <see cref="meta"/> API.</param>
         /// <seealso href="@overriding-fields-or-properties"/>
-        IOverrideAdviceResult<IProperty> OverrideAccessors(
+        IOverrideAdviceResult<IPropertyOrIndexer> OverrideAccessors(
```

Comment:
Would it make sense to have convenience overloads for `IOverrideAdviceResult<IProperty> OverrideAccessors(IFieldOrProperty, ...)` and `IOverrideAdviceResult<IIndexer> OverrideAccessors(IIndexer, ...)`?

The same for `AddContract`.

(This would also make the breaking change less serious.)

Reply:
The breaking change affects only the return value, which was nonsensical anyway. I'd leave that to Gael's Public API review when we are merging the main story.